### PR TITLE
Stepper: Catch and show "name not declared" error 

### PR DESCRIPTION
**Pull request by hanscau**
Fix for #1551 

Catch RuntimeSourceError and display it on the stepper

Add appropriate test for "name not declared"

### CODE REVIEW

File: src/stepper/__tests__/stepper.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ function getLastStepAsString(steps: [substituterNodes, string[][], string][]): s
   return codify(steps[steps.length - 1][0]).trim()
 }
 
+function getExplanation(steps: [substituterNodes, string[][], string][]): string {
+  return steps.map(x => x[2]).join('\n')
```
Comment (RichDom2185) [inline src/stepper/__tests__/stepper.ts:13]: What is the `[2]` here? Can we add a comment?

**Review by RichDom2185 — COMMENTED** [review]
Just one minor readability/code clarity nit below:

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

File: src/stepper/__tests__/stepper.ts (hunk 1 above)
Comment (hanscau) [reply src/stepper/__tests__/stepper.ts:14]: Left a comment for now, in the future all the indexes of steps should be accessed through a constant. Or maybe steps should just be an object array instead for better clarity...

**Pull request merged**